builder/dockerfile/containerbackend.go

```diff
@@ -93,6 +94,7 @@ func (c *containerManager) Run(ctx context.Context, cID string, stdout, stderr i
 		close(finished)
 		logCancellationError(cancelErrCh,
 			fmt.Sprintf("a non-zero code from ContainerWait: %d", status.ExitCode()))
+		err = errutils.WrapNil(err, status.Err())
```

Comment:
In what case would `err` be non-nil? I can't see any.

Reply:
Yes, you're right here. Will update.